Makefile

```diff
@@ -175,15 +185,17 @@ $(CONTROLLER_GEN): $(LOCALBIN)
 	GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-tools/cmd/controller-gen@$(CONTROLLER_TOOLS_VERSION)
 
 .PHONY: image
-build-image: ## Build the kro controller images using ko build
-	$(WITH_GOFLAGS) KOCACHE=$(KOCACHE) \
-		ko build --bare github.com/kro-run/kro/cmd/controller \
+build-image: ko ## Build the kro controller images using ko build
+	echo $(RELEASE_VERSION)
+	$(WITH_GOFLAGS) KOCACHE=$(KOCACHE) KO_DOCKER_REPO=$(KO_DOCKER_REPO) \
+		$(KO) build --bare github.com/kro-run/kro/cmd/controller \
+		--local\
```

Comment:
QQ about the `--local` flag in build-image rule for publishing official kro images - can this affect our build/release pipeline? https://github.com/kro-run/kro/blob/main/.github/workflows/build-push-image.yaml#L62-L66 (we don't have any docker daemon running in those jobs AFAIK)

Reply:
It is optional. The local is required to work with kind. 

Doesnt affect the build-push image workflow: https://github.com/kro-run/kro/actions/runs/13280535130/job/37077790345?pr=279